tests/cases/standalone/delete/delete_invalid.result

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE monitor ( host STRING, ts TIMESTAMP, cpu DOUBLE DEFAULT 0, memory DOUBLE, TIME INDEX (ts), PRIMARY KEY(host)) ;
+
+Affected Rows: 0
+
+insert into monitor(host, cpu, memory, ts) values ('host1', 66.6, 1024, 1655276557000), ('host2', 77.7,  2048, 1655276558000), ('host3', 88.8,  3072, 1655276559000);
+
+Affected Rows: 3
+
+delete from monitor where cpu = 66.6 and ts = 1655276557000;
+
+Error: 1004(InvalidArguments), Missing column host in write batch
+
+delete from monitor where host = 'host1' or ts = 1655276557000;
+
+Error: 1004(InvalidArguments), Invalid SQL, error: Failed to parse expr:host = 'host1' OR ts = 1655276557000
+
+delete from monitor where host = 'host1' or ts != 1655276557000;
+
+Error: 1004(InvalidArguments), Invalid SQL, error: Failed to parse expr:host = 'host1' OR ts <> 1655276557000
```

Comment:
Those error messages look ambiguous. SQLs do not fail due to grammar errors (indicated by "failed to parse"), but they don't provide correct delete conditions.